dahbug.php

```diff
@@ -955,7 +955,7 @@ static protected function _colorize($string, $color = null)
             return $string;
         }
 
-        $string = "\033[{$colorNumber}m" . $string . "\033[39m";
+        $string = "\033[{$colorNumber}m" . $string . "\033[0m";
```

Comment:
Thank you for pointing this out.
The problem with this is that background color will also be reset. I've changed the escape code to "\033[39;0m" which will reset bold state aswell. Let me know if this does the trick.